app/src/main/res/layout/fragment_news.xml

```diff
@@ -1,6 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+
+<androidx.swiperefreshlayout.widget.SwipeRefreshLayout android:id="@+id/vSwipeRefreshLayout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical"/>
\ No newline at end of file
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:id="@+id/vFragmentNews">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/vRecyclerViewNews"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginStart="0dp"
+        android:layout_marginTop="0dp"
+        android:scrollbars="vertical" />
+
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/fab_icon"
+        android:layout_width="90dp"
+        android:layout_height="74dp"
+        android:layout_below="@+id/vRecyclerViewNews"
+        android:layout_alignParentStart="true"
+        android:layout_marginStart="336dp"
```

Comment:
It would be better to set gravity right bottom.

Reply:
I solve this using `layout_alignParentBottom` `layout_marginEnd` `layout_marginBottom` because gravity didn't work inside `SwipeRefreshLayout`